### PR TITLE
tidb-in-kubernetes: add cluster operation guide

### PR DESCRIPTION
**Pull request by DanielZhangQD**
- Related PR: 
  - https://github.com/pingcap/docs-cn/pull/1497
  - https://github.com/pingcap/docs-cn/pull/1582
  - https://github.com/pingcap/docs-cn/pull/1552
  - https://github.com/pingcap/docs-cn/pull/1585/
- Preview: https://github.com/DanielZhangQD/docs/blob/operation-migration/tidb-in-kubernetes/operation-guide.md

### CODE REVIEW

File: tidb-in-kubernetes/operation-guide.md [hunk 1]
```diff
@@ -0,0 +1,252 @@
+# TiDB Cluster Operation Guide
```
Comment (lilin90) [inline tidb-in-kubernetes/operation-guide.md]: Please add the metadata.

Comment (DanielZhangQD) [reply tidb-in-kubernetes/operation-guide.md]: done

**Comment by lilin90** [comment]
@DanielZhangQD This document was updated by @xiaojingchen in https://github.com/pingcap/docs-cn/pull/1585. Please update it here. Thanks!

**Comment by DanielZhangQD** [comment]
> @DanielZhangQD This document was updated by @xiaojingchen in [pingcap/docs-cn#1585](https://github.com/pingcap/docs-cn/pull/1585). Please update it here. Thanks!

@lilin90 The change refers to cluster-configuration.md which is not in docs repo yet, I think we should not update the change now.

**Comment by lilin90** [comment]
@DanielZhangQD Thanks for your reminder. You can omit that sentence then. When that file is added, we add that link back.

**Comment by DanielZhangQD** [comment]
@lilin90 New commit of porting tidb-operator#612 and docs-cn#1585.

**Pull request merged**